core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -120,112 +150,108 @@ oppia.controller('ExplorationHistory', [
 
   // Function to set compared version metadata, download YAML and generate
   // diff graph and legend when selection is changed
-  $scope.changeCompareVersion = function(versionNumber, changedVersion) {
+  $scope.changeCompareVersion = function(versionNumber) {
```

Comment:
It wasn't being used in the original version either, so I didn't think to change it. Will remove it now.